ext/hash/php_hash_sha.h

```diff
@@ -56,10 +56,10 @@ PHP_HASH_API void PHP_SHA256Update(PHP_SHA256_CTX *, const unsigned char *, size
 void SHA256_Transform_sse2(uint32_t state[PHP_STATIC_RESTRICT 8], const uint8_t block[PHP_STATIC_RESTRICT 64], uint32_t W[PHP_STATIC_RESTRICT 64], uint32_t S[PHP_STATIC_RESTRICT 8]);
 #endif
 
-#if (defined(__i386__) || defined(__x86_64__)) && defined(HAVE_IMMINTRIN_H)
+#if (defined(__i386__) || defined(__x86_64__)) && defined(HAVE_IMMINTRIN_H) || defined(_M_X64) || defined(_M_IX86)
```

Comment:
Should have explicit parentheses to make precedence between `&&` and `||` clear.